src/components/Organizations/Card.tsx

```diff
@@ -64,7 +64,7 @@ const OrganizationCardSkeleton = ({ electionCount: ec, ...rest }: IOrganizationC
           ) : (
             <OrganizationName fontWeight={'bold'} wordBreak='break-all' size='sm' />
           )}
-          <ReducedTextAndCopy color={'textAccent1'} toCopy={pid}>
+          <ReducedTextAndCopy reduced={true} color={'textAccent1'} toCopy={pid}>
```

Comment:
I counted four more occurrences (no need to review each one IMO)

Reply:
Reduced is not used anymore 62cb7e5